tests/test_io.py

```diff
@@ -23,28 +26,115 @@ def captured_output():
 
 
 class Test_load_program_texts():
-    def test_load_program_texts_01(self):
+     def test_load_program_texts_in_global_PROGTEXT(self):
```

Comment:
We are not doing given when then?

Reply:
As discussed with @MRVermeulenDeltares, this should be resolved in [DFAST-107](https://issuetracker.deltares.nl/browse/DFAST-107)